packages/opensrp-plans/src/helpers/utils.tsx

```diff
@@ -0,0 +1,49 @@
+import { Dictionary } from '@onaio/utils';
+import { InterventionType } from '../ducks';
+import { PlanDefinition, UseContext } from '../plan-global-types';
+
+/**
+ * helper to retrieve the plan Type from a plan definition object
+ *
+ * @param {PlanDefinition} aPlan plan definition object
+ * @returns {string} plans intervention type eg FI
+ */
+export const getPlanType = (aPlan: PlanDefinition) => {
+  return aPlan.useContext
+    .filter((f: UseContext) => f.code === 'interventionType')
+    .map((context) => context.valueCodableConcept)[0];
+};
+
+/**
+ * Sorts plans in descending order based on field provided
+ *
+ * @param {PlanDefinition[]} arr plan definition array
+ * @param {string} sortField Field to sort by
+ * @returns {PlanDefinition[]}  sorted plans
+ */
+export function descendingOrderSort<T extends object>(arr: T[], sortField: string) {
```

Comment:
Are we only ever interested in descending order sorting?

Reply:
Sure, we will eventually require both descending and ascending order sorting. Hope it's fine to create an issue for this

Reply:
https://github.com/OpenSRP/web/issues/204